### ISSUE TITLE
Bad Training Results Using the Darknet_conv. Is there some initial configuration that I need to change in order to get better results? 

### ISSUE THREAD

**Issue by Rune1o618** [Microsoft/singleshotpose#190]
python train.py --datacfg cfg/ape.data --modelcfg cfg/yolo-pose.cfg --initweightfile cfg/darknet19_448.conv.23 --pretrain_num_epochs 15

Using this command for training the model gives really bad accuracy? Can someone please explain to to pre-train the model properly?
